pca_cca.go

```diff
@@ -57,13 +57,12 @@ func (c *PC) Vectors(dst *mat64.Dense) *mat64.Dense {
 		panic("stat: use of unsuccessful principal components analysis")
 	}
 
-	if dst == nil {
-		dst = &mat64.Dense{}
-	} else if d, n := dst.Dims(); (n != 0 || d != 0) && (d != c.d || n != min(c.n, c.d)) {
-		panic(matrix.ErrShape)
+	if dst != nil {
+		if d, n := dst.Dims(); (n != 0 || d != 0) && (d != c.d || n != min(c.n, c.d)) {
```

Comment:
:) This is an intentionally left sore to provoke the exporting of `IsZero` in mat64.